src/middlewared/middlewared/plugins/system.py

```diff
@@ -1444,6 +1456,16 @@ async def _event_system(middleware, event_type, args):
         # If it is not defined yet, it will try to define
         if birthday is None:
             asyncio.ensure_future(_update_birthday(middleware))
+
+        if osc.IS_LINUX and not FIRST_BOOT and (await middleware.call('system.advanced.config'))['kdump_enabled']:
+            cp = await run(['kdump-config', 'status'], check=False)
+            if cp.returncode:
+                middleware.logger.error('Failed to retrieve kdump-config status: %s', cp.stderr.decode())
+            else:
+                if not RE_KDUMP_CONFIGURED.findall(cp.stdout.decode()):
+                    await middleware.call('alert.oneshot_create', 'KdumpNotReady', None)
+                else:
+                    await middleware.call('alert.oneshot_delete', 'KdumpNotReady', None)
```

Comment:
What if we had the alert, disabled kdump and rebooted? It will stay forever in this case.